### ISSUE TITLE
Mouse cursor incorrectly warped when opening floating window on different output

### ISSUE THREAD

**Issue by skontar** [i3/i3#2681]
Output of `i3 --moreversion 2>&- || i3 --version`:

```
Binary i3 version:  4.13-91-gb03412d (2017-01-25, branch "gaps-next") © 2009 Michael Stapelberg and contributors
Running i3 version: 4.13-91-gb03412d (2017-01-25, branch "gaps-next") (pid 12401)
Loaded i3 config: /home/skontar/.config/i3/config (Last modified: Sun 12 Feb 2017 11:29:38 PM CET, 62300 seconds ago)

The i3 binary you just called: /usr/bin/i3
The i3 binary you are running: i3-with-/i3
```

URL to a logfile as per http://i3wm.org/docs/debugging.html:

http://logs.i3wm.org/logs/5740615169998848.bz2

**What I did:**

- Opened Firefox - it is going to workspace "4:  " on screen nr2 DP2-1.
- Firefox gets focus.
- Click place on the i3bar running i3blocks which starts `gsimplecal` on screen nr1 (primary) DP2-2.

**What I saw:**

- gsimplecal opens on screen nr1 (primary) DP2-2 and it is focused as per my config visible in log.
- mouse cursor is warped back to nr2 DP2-1, even when gsimplecal is still focused.

**What I expected instead:**

- mouse cursor stays on screen nr1 (primary) DP2-2.

**Comment by skontar** [comment]
It seems to be similar to https://github.com/i3/i3/issues/2592 but I am not quite sure that it is the same. Let me know if you need any more data or logs, I can try to reproduce it in VM with clean i3 install if needed.

**Comment by skontar** [comment]
So actually the following log should be even better and conditions easier to reproduce:
http://logs.i3wm.org/logs/5648269346603008.bz2

I have used default config, only added following lines:
```
focus_follows_mouse no
bindsym F12                    exec xfce4-terminal --drop-down
```

I have moved cursor to screen 2, opened a terminal there. Then moved cursor to screen 1, and hit F12. Dropdown terminal appears correctly on screen 1 and gets focus, cursor goes back to screen 2. 

Hope that helps.
